tests/test_utils.py

```diff
@@ -145,13 +145,45 @@ def test_not_authorizated_group(self):
                 with self.assertRaises(DSSForbiddenException):
                     security.assert_authorized_group(['hca'], token_info)
 
+    @mock.patch('dss.Config._OIDC_AUDIENCE', new=["https://dev.data.humancellatlas.org/",
+                                                  "https://data.humancellatlas.org/"])
+    @mock.patch('dss.Config._TRUSTED_GOOGLE_PROJECTS', new=['cool-project-188401.iam.gserviceaccount.com'])
+    def test_verify_jwt_multiple_audience(self):
+        jwts_positive = [
+            get_service_jwt(UNAUTHORIZED_GCP_CREDENTIALS),
+            get_service_jwt(UNAUTHORIZED_GCP_CREDENTIALS, audience="https://dev.data.humancellatlas.org/"),
+            get_service_jwt(UNAUTHORIZED_GCP_CREDENTIALS, audience="https://data.humancellatlas.org/"),
+            get_service_jwt(UNAUTHORIZED_GCP_CREDENTIALS, audience=["https://dev.data.humancellatlas.org/", "e"])
```

Comment:
_e_ is a positive case?

Reply:
at least one of the member in the list must be in our list of approved. 'e' is not but 'dev.data...' is.